### PR TITLE
Removing "ti update" as it does not exist (anymore)

### PR DESCRIPTION
**Pull request by FokkeZB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Dependency Status](https://david-dm.org/appcelerator/titanium.svg)](https://david-dm.org/appcelerator/titanium)
 [![devDependency Status](https://david-dm.org/appcelerator/titanium/dev-status.svg)](https://david-dm.org/appcelerator/titanium#info=devDependencies)
 [![NPM version](https://badge.fury.io/js/titanium.svg)](http://badge.fury.io/js/grunt-appc-js)
-
+u
```
Comment (cb1kenobi) [inline README.md]: What's up with u?

**Comment by FokkeZB** [comment]
Huh? That wasn't my change. Something went wrong there. Should have removed the "Ti SDK update" section

**Comment by cb1kenobi** [comment]
The commit says otherwise. Remove the 'u' and we'll get this merged.

**Comment by FokkeZB** [comment]
I've removed u ;)

**Comment by cb1kenobi** [comment]
Another masterpiece! **APPROVED**

**Pull request merged**